include/tss2/tss2_tcti_spi.h

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 Fraunhofer SIT. All rights reserved.
+ */
+#ifndef TSS2_TCTI_SPI_H
+#define TSS2_TCTI_SPI_H
+
+#include "tss2_tcti.h"
+#include <driver/spi_master.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define TSS2_TCTI_SPI_PLATFORM_MAGIC 0x61F7D74109DB0D8BULL
+
+typedef struct TSS2_TCTI_SPI_PLATFORM TSS2_TCTI_SPI_PLATFORM;
+typedef void (*TSS2_TCTI_SPI_PLATFORM_SLEEP_MS_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform, int milliseconds);
+typedef void (*TSS2_TCTI_SPI_PLATFORM_START_TIMEOUT_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform, int milliseconds);
```

Comment:
All of these functions need some more documentation.
What's the service expected from this function ?

Reply:
More documentation has been added in the header file and in doc/tcti-spi.md